api/src/main/java/com/sevenbridges/apiclient/task/ExecutionStatus.java

```diff
@@ -76,4 +76,52 @@
    */
   Integer getAborted();
 
+  /**
+   * Boolean indicating whether the platform instance limit has bean reached.
+   *
+   * @return Boolean system limit
+   */
+  Boolean getSystemLimit();
+
+  /**
+   * Boolean indicating whether the account instance limit has bean reached.
+   *
+   * @return Boolean account limit
+   */
+  Boolean getAccountLimit();
+
+  /**
+   * Boolean indicating whether the task is in initializing state.
+   *
+   * @return Boolean instance init
+   */
+  Boolean getInstanceInit();
+
+  /**
+   * Long amount of time that task has spent in the Queued state.
+   *
+   * @return Long queued duration
+   */
+  Integer getQueuedDuration();
```

Comment:
Make method name unambiguous, no way to determine time unit on this API, and because only number is returned, it should have suffix describing the unit, in plural form.
Suggestion: `getQueuedDurationSeconds()` (or whatever the unit is; should probably be the same for other ones).